src/main/java/com/philips/research/bombase/core/nuget/domain/NugetAPI.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (c) 2020-2021, Koninklijke Philips N.V., https://www.philips.com
+ * SPDX-License-Identifier: MIT
+ */
+
+package com.philips.research.bombase.core.nuget.domain;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.philips.research.bombase.core.meta.PackageMetadata;
+import com.philips.research.bombase.core.meta.registry.Field;
+import pl.tlinkowski.annotation.basic.NullOr;
+import retrofit2.Call;
+import retrofit2.http.GET;
+import retrofit2.http.Path;
+
+import java.net.URI;
+import java.util.Date;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public interface NugetAPI {
```

Comment:
Suggestion to add a JavaDoc comment indicating the site where the supported API is specified (or what we used to reverse-engineer the format), so maintainers don't have to look it up themselves.